src/CodeGeneration.Roslyn.Tool/Program.cs

```diff
@@ -53,8 +53,10 @@ static int Main(string[] args)
             {
                 generator.Generate(progress);
             }
-            catch
+            catch (Exception e)
             {
+                Logger.Log(LogLevel.High, $"{e.Message}: {e.Message}");
```

Comment:
You're emitting the exception message twice. Did you mean to log the exception type for the first one?